### ISSUE TITLE
[Question] Other Language files

### ISSUE THREAD

**Issue by Dream-Master** [yesterday17/GTNH-GuideBook-i18n#11]
@yesterday17 is it possible with the lastest script file and a German file (we create the file) to use it ingame ? or the original us english version too? 
I like to integrate it in the next update as standart.

**Comment by Yesterday17** [comment]
Yes. BetterQuesting supports it. English version is necessary because it's the default language file if specific one doesn't exist, but if it exists, it will use the correct language file.
So just rename the file you get by the script (in this repository it should be at /assets/minecraft/lang/NU_LL.lang) to 'de_DE.lang', and add translations to it. It should work well.

Also, i wrote a the 'update' function a few hours before. You can get a result of comparison between an older version and a newer version of the tasks. Detailed guides are written in /scripts/Readme.md.
